akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -928,7 +928,7 @@ trait LoggingFSM[S, D] extends FSM[S, D] { this: Actor =>
         case a: ActorRef             => a.toString
         case _                       => "unknown"
       }
-      log.debug("processing {} from {} in state {}", event, srcstr, stateName)
+      log.debug("{} is processing {} from {} in state {}", self, event, srcstr, stateName)
```

Comment:
This shouldn't be needed. The path of the logging actor is always included. When using SLF4J it's a mdc value `akkaSource`, see https://doc.akka.io/docs/akka/current/logging.html#logging-thread-akka-source-and-actor-system-in-mdc

Reply:
Have not know about this, thanks a lot. I'm closing the issue then.